tests/test_models/test_layers/test_plugins.py

```diff
@@ -1,185 +0,0 @@
-# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
This file was deleted in PR #9089. It should have been added back in PR #9515. But I miss it.
This does not affect the merging of the branch, since both the training and inference experiments of MaskFormer and Mask2Former have been completed.
I can create a new PR to add it back after branch mergering.

Reply:
The conversation has been resolved in PR #9656.